web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -142,17 +157,33 @@ const TeamReviews = ({ onBack, periodId }) => {
   const reviewAssignmentsUrl = '/services/review-assignments';
 
   useEffect(() => {
-    loadTeamMembers();
```

Comment:
I renamed this function and moved some of its functionality to a new function named `loadTeamMembers`.
This allows `loadTeamMembers` to also be called in another place.